net-im/ejabberd/ejabberd-21.04.ebuild

```diff
@@ -0,0 +1,292 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+SSL_CERT_MANDATORY=1
+
+inherit eutils pam rebar ssl-cert systemd tmpfiles
```

Comment:
Is eutils used here? Need EAPI 6?

Reply:
Good catch, `eutils` does not seem to be used. What do you mean with "Need EAPI 6"?

Reply:
Always worth auditing eclasses! Especially eutils which is one of those horrible ones which creeps up a lot unnecessarily. multillib (no suffix) is similar as it was in the past needed for get_libdir.

Re EAPI, I was just asking if we couldn’t do 7.